book/team.yaml

```diff
@@ -510,4 +510,26 @@ people:
         icon_pack: fab
         link: https://github.com/mlwrzesien
     user_groups:
-      - Tutorial Lead
\ No newline at end of file
+      - Tutorial Lead
+  # =============
+  - title: Xiaolan Xu	
+    avatar: https://avatars.githubusercontent.com/u/7988251?v=4
+    role: Tutorial Lead
+    organizations:
+    - name: Jet Propulsion Lab
+    - name: Radar Science
+    bio: >
+      Expertise in the field of applied electromagnetics, electromagnetic wave propagation and scattering properties from snow-covered terrain, 
+      vegetated land surface, and bare soil, forward modeling, and retrieval algorithm development with applications to Earth remote sensing 
+      in space. She currently leads the NASA Earth Science Research project to improve snow water equivalence estimate using radar 
+      tomography technique and serves on the Science Team for NASA Soil Moisture Active and Passive Mission and Ku-band Radar mission 
+      supported by Canadian Space Agency.
+    expertise:
+    - Microwave Remote Sensing
+    - Signals of Opportunity (SoOP)
+    social:
+      - icon: github
+      icon_pack: fab
+      link: https://github.com/xlxu220
+    user_groups:
+      - SoOP Tutorial Lead
```

Comment:
@xlxu220 this line is failing the spelling check. Can you remove "SoOP"? The user groups are unique names so keeping it consistent with the others is best. Thanks!

Reply:
to skip spellcheck words or acronyms you can also add them to this list https://github.com/snowex-hackweek/website2022/blob/29fc416ec7a24d5d602b0773ed0706b02cf76afe/.github/workflows/qaqc.yaml#L36